### PR TITLE
Expose Request and RequestMethods to public

### LINKED ISSUE #11: Expose Request and RequestMethods types from the package
# Description

Right now it is necessary to expose them (make them part of the public API) in order to make `onRoute()` calls by providing an optional request instance.

### PR DESCRIPTION
**Pull request by GiorgiBeriashvili**
Resolves #11

### CODE REVIEW

File: lib/src/handlers/mock_handler.dart [hunk 1]
```diff
@@ -16,7 +15,7 @@ class MockHandler {
   Future<ResponseBody> createMock(String fullPath, dynamic data) async {
```
Comment (LukaGiorgadze) [inline lib/src/handlers/mock_handler.dart:15]: add params in comment:
`/// Creates and writes the mock data in [fullPath] inside a file as a JSON object [data].`

**Review by LukaGiorgadze — CHANGES_REQUESTED** [review]

File: lib/src/handlers/mock_handler.dart (hunk 1 above)
Comment (GiorgiBeriashvili) [reply lib/src/handlers/mock_handler.dart:17]: შესწორებულია, ოდნავ გადავაკეთე, `readMock`-იც.

**Review by LukaGiorgadze — APPROVED** [review]
ჩემგან okay, დაველოდოთ ნატალის

**Review by LukaGiorgadze — APPROVED** [review]

**Comment by GiorgiBeriashvili** [comment]
> ჩემგან okay, დაველოდოთ ნატალის

Had to edit the naming as well due to incompatibility, re-review requested.

And whoops, edited your comment by mistake. Fixed.

**Comment by nataliehippe** [comment]
good for go

**Pull request merged**